stdlib/2and3/xml/etree/ElementTree.pyi

```diff
@@ -122,11 +118,34 @@ class ElementTree:
 
 def register_namespace(prefix: _str_argument_type, uri: _str_argument_type) -> None: ...
 if sys.version_info >= (3, 4):
-    def tostring(element: Element, encoding: Optional[str] = ..., method: Optional[str] = ..., *, short_empty_elements: bool = ...) -> _tostring_result_type: ...
-    def tostringlist(element: Element, encoding: Optional[str] = ..., method: Optional[str] = ..., *, short_empty_elements: bool = ...) -> List[_tostring_result_type]: ...
+    @overload
+    def tostring(element: Element, encoding: None = ..., method: Optional[str] = ..., *, short_empty_elements: bool = ...) -> bytes: ...
+    @overload
+    def tostring(element: Element, encoding: Literal["unicode"], method: Optional[str] = ..., *, short_empty_elements: bool = ...) -> str: ...
+    @overload
+    def tostring(element: Element, encoding: str, method: Optional[str] = ..., *, short_empty_elements: bool = ...) -> Union[bytes, str]: ...
```

Comment:
This should return `Any`. `Union` return types are discouraged, since they mean that a called always has to use `isinstance`.